src/settings/OpeningPeriodForm/OpeningPeriodFormWrapper.js

```diff
@@ -123,13 +123,18 @@ class OpeningPeriodFormWrapper extends Component {
     });
   }
 
-  catchOverlappedEvents = async err => {
-    if (err.status === 422) {
+  catchOverlappedEvents = async (err) => {
+    if (err.status >= 400) {
       const response = await err.json();
-      const errorMessage = response.errors[0].message;
+      const error = response.errors[0];
 
       this.setState({
-        errorModalText: errorMessage,
+        errorModalText: (
+          <FormattedMessage
+            id={`ui-calendar.${error.code}`}
```

Comment:
Translation `"intervalsOverlap": "Intervals can not overlap.",` used for Exception Period. Could you please discuss with product owner should we use this translation for current case or we need create new one?

Reply:
Asked on JIRA: https://issues.folio.org/browse/UICAL-193?focusedCommentId=118853&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-118853

Reply:
Hello @ncovercash Could you please discuss with product owner what text we need to use for this error
`The date range entered overlaps with another calendar for this service point. Please correct the date range or enter the hours as exceptions.` - error from BE or `Intervals can not overlap` - used for Exception Period or we need create new one?

Reply:
@Dmitriy-Litvinenko I asked the product owner for comment on the JIRA link above, however, have yet to receive a response.  I'll let you know here as soon as I get one.